ziti-tunnel/docker/docker-entrypoint.sh

```diff
@@ -3,10 +3,13 @@
 set -euo pipefail
 
 function alldone() {
-    # send SIGINT to ziti-tunnel to trigger a cleanup of iptables mangle rules
-    kill -INT $ZITI_TUNNEL_PID
-    # let entrypoint script exit after ziti-tunnel PID
-    wait $ZITI_TUNNEL_PID
+    # if successfully sent to background then send SIGINT to trigger a cleanup
+    # of iptables mangle rules and loopback assignments
+    [[ "${ZITI_TUNNEL_PID:-}" =~ ^[0-9]+$ ]] && {
+        kill -INT "$ZITI_TUNNEL_PID"
+        # let entrypoint script exit after ziti-tunnel PID
+        wait "$ZITI_TUNNEL_PID"
```

Comment:
The conditional is necessary for this trap function because `ZITI_TUNNEL_PID` is not defined if the entrypoint script encounters an error and calls this function prior to sending `ziti-edge-tunnel` to the background. This leads to unset var errors when `ZITI_TUNNEL_PID` is expanded for the `kill` command.